src/modules/imageresizer/ImageResizerContextMenu/Resources.resx

```diff
@@ -118,7 +118,8 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Resize_Pictures_Title" xml:space="preserve">
-    <value>Resize pictures</value>
+    <value>Resize with Image Resizer</value>
+    <comment>'Image Resizer' is a product name, do not loc.</comment>
```

Comment:
It is translated in most languages, so I wouldn't add this comment here.

Reply:
Oh really.. thought we didn't translate module names 😲

Reply:
> Oh really.. thought we didn't translate module names 😲

Only for specific names like "FancyZones" or "PowerRename"

![image](https://github.com/microsoft/PowerToys/assets/58633848/d3aab81e-8d28-4cca-b0d5-bc5b296f3d0d)

Reply:
Thanks, updated!